fdb-relational-core/src/main/antlr/RelationalLexer.g4

```diff
@@ -127,6 +127,8 @@ INSERT:                              'INSERT';
 INTERVAL:                            'INTERVAL';
 INTO:                                'INTO';
 IS:                                  'IS';
+IS_DISTINCT_FROM:                    'IS DISTINCT FROM';
+IS_NOT_DISTINCT_FROM:                'IS NOT DISTINCT FROM';
```

Comment:
This is unusual, you should split the tokens to enable tokenizing them correctly disregarding whitespaces.

Reply:
got it, modified.